test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-update/resource-assert.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: image-secret
+  namespace: prod
```

Comment:
Because this is a clone, I would recommend that we assert for the entire contents of the generated resources and not just their existence. We want to ensure they exist but they exist with the expected contents. This wouldn't do that.

Reply:
Makes sense, we should add the entire content in earlier merged generate test as well. Will fix them in upcoming PRs.
I will push the update for this tests.

Reply:
fixed